src/app/containers/Metadata/index.jsx

```diff
@@ -225,7 +183,7 @@ const MetadataContainer = ({ metadata, promo }) => {
         articleAuthor={articleAuthor}
         articleSection={pathOr(null, ['passport', 'genre'], metadata)}
         brandName={brandName}
-        canonicalLink={canonicalLink}
+        canonicalLink={canonicalNonUkLink}
```

Comment:
I believe this changes the canonical link for `/news/articles/*` pages to always use `*bbc.com`. I don't think that is what we want. 
```suggestion
        canonicalLink={canonicalLink}
```

Reply:
Seems I was mistaken. Canonical is pinned to use `www.bbc.com`/ `www.test.bbc.com`